src/testIntegration/java/uk/gov/justice/digital/delius/service/AlfrescoServiceTest.java

```diff
@@ -37,14 +37,18 @@ public void shouldBeNotFoundIfDocumentDoesNotBelongToOffender() {
 
     @Test
     public void shouldReturnResourceWhenFound() throws IOException {
-        AlfrescoExtension.alfrescoMockServer.stubDetailsSuccess("123", "T1234", "document.pdf");
+
+        String filename = "document (1).pdf";
+
+        AlfrescoExtension.alfrescoMockServer.stubDetailsSuccess("123", "T1234", filename);
         AlfrescoExtension.alfrescoMockServer.stubFetchDocument("123", new byte[]{'a', 'b', 'c'});
 
         final var response = alfrescoService.getDocument("123", "T1234");
 
         assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
         assertThat(response.getBody().contentLength()).isEqualTo(3);
-        assertThat(response.getHeaders().getFirst(HttpHeaders.CONTENT_DISPOSITION)).isEqualTo("attachment; filename=\"document.pdf\"");
+        assertThat(response.getHeaders().getFirst(HttpHeaders.CONTENT_DISPOSITION)).isEqualTo(ContentDisposition.attachment()
+            .filename(filename, StandardCharsets.UTF_8).build().toString());
```

Comment:
Tiny nitpick - would it make more sense to assert against the actual expected string?  Just to make it clear that the values will be encoded.
```suggestion
        assertThat(response.getHeaders().getFirst(HttpHeaders.CONTENT_DISPOSITION))
            .isEqualTo("attachment; filename*=UTF-8''document%20%281%29.pdf");
```